concepts/05 UI Components/List/00 Getting Started with List/03 Bind the List to Data.md

```diff
@@ -0,0 +1,774 @@
+The **List** can load data from different data source types. To use a local array, assign it to the [dataSource](Documentation/ApiReference/UI_Components/dxList/Configuration/#dataSource) property and use the [displayExpr](/Documentation/ApiReference/UI_Components/dxList/Configuration/#displayExpr) property to specify the name of the data field whose values you would like to display. For information on data binding with other data source types, refer to the following articles:
+
+- [Read-Only Data in JSON Format](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Read-Only_Data_in_JSON_Format/)
+- [OData](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/OData/)
+- [Web API, PHP, MongoDB](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Web_API,_PHP,_MongoDB/)
+- [Custom Data Sources](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Custom_Data_Sources/)
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function () {
+        $('#list').dxList({
+            dataSource: products,
+            displayExpr: 'Name',
+        });
+    });
+
+    <!-- tab: data.js -->
+    var products = [{
+        ID: 1,
+        Name: 'HD Video Player',
+        Price: 330,
+        Current_Inventory: 225,
+        Backorder: 0,
+        Manufacturing: 10,
+        Category: 'Video Players',
+        ImageSrc: 'images/products/1.png',
```

Comment:
Let's edit the data source to exclude those fields that we don't use in this tutorial.